### PR TITLE
Tpch in Spark Cluster branch

### PR DESCRIPTION
**Pull request by DavideTrupia**
#5

### CODE REVIEW

File: benchmarking/cluster_tpch_execution_times.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Spark.executor.memory 6GB
+Spark.executor.cores 3
```
Comment (ssavvides) [inline benchmarking/cluster_tpch_execution_times.txt:2]: Is spark running in docker here?

What's the input size? ~1GB?

**Review by ssavvides — APPROVED** [review]

File: benchmarking/cluster_tpch_execution_times.txt (hunk 1 above)
Comment (DavideTrupia) [reply benchmarking/cluster_tpch_execution_times.txt:2]: Yes, should i add it to the file?

Comment (ssavvides) [reply benchmarking/cluster_tpch_execution_times.txt:2]: Yes add a note pointing to the dockerfiles used to generate the images

Also mention the size of the input

Comment (ssavvides) [reply benchmarking/cluster_tpch_execution_times.txt:2]: if you just used the command`./dbgen` to generate the files, the total input is about 1GB across all tables

Comment (DavideTrupia) [reply benchmarking/cluster_tpch_execution_times.txt:9]: Yes, I generated from dbgen. Added in file

**Pull request merged**